app/assets/javascripts/models/tour.js

```diff
@@ -0,0 +1,51 @@
+import Tether from 'tether';
+import Shepherd from 'tether-shepherd';
+
+module.exports = Backbone.Model.extend({
+  name: 'tour',
+
+  initialize: function() {
+    this.shepherd = new Shepherd.Tour({
+      defaults: {
+        classes: 'shepherd-theme-arrows',
+        scrollTo: false
+      }
+    });
+  },
+
+  start: function() {
+    this.shepherd.start();
+  },
+
+  next: function() {
+    this.shepherd.next();
+  },
+
+  complete: function(user) {
+    this.finishTour(user);
```

Comment:
Why not just simply receive the user on the constructor instead of passing it around?